### PR TITLE
add custom pin annotation and start custom callout

### PR DESCRIPTION
**Pull request by sokolikp**
Custom callout is not done yet. I created a nib for it, but I'm having trouble connecting it to the callout. I might need to pair with one of you to get this set up.
screenshot 2017-10-26 20 40 45

### CODE REVIEW

File: Sweeper.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -9,6 +9,9 @@
 /* Begin PBXBuildFile section */
 		374C23E81F91BDA5000C0ECE /* Pin.swift in Sources */ = {isa = PBXBuildFile; fileRef = 374C23E71F91BDA5000C0ECE /* Pin.swift */; };
 		374C23EA1F91BDAE000C0ECE /* User.swift in Sources */ = {isa = PBXBuildFile; fileRef = 374C23E91F91BDAE000C0ECE /* User.swift */; };
+		3771ABBF1FA1A33A00A06F17 /* PinCalloutView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3771ABBE1FA1A33A00A06F17 /* PinCalloutView.swift */; };
+		3771ABC41FA1AA0300A06F17 /* CustomPinCalloutView.xib in Resources */ = {isa = PBXBuildFile; fileRef = 3771ABC31FA1AA0300A06F17 /* CustomPinCalloutView.xib */; };
```
Comment (wumingx89) [inline Sweeper.xcodeproj/project.pbxproj]: I would name them the same just for convention/consistency

Comment (wumingx89) [inline Sweeper.xcodeproj/project.pbxproj:13]: What's the difference between the PinCalloutView and CustomCalloutView?

**Review by wumingx89 — APPROVED** [review]
LGTM, just had a question on some of the files added

**Comment by sokolikp** [comment]
Merging for now. We can rename files in cleanup.

**Pull request merged**